src/battle-tooltips.ts

```diff
@@ -1470,6 +1470,10 @@ class BattleTooltips {
 		if (this.battle.hasPseudoWeather('Gravity')) {
 			value.modify(5 / 3, "Gravity");
 		}
+		// 1/256 glitch
+		if (this.battle.gen === 1 && !toID(this.battle.tier).includes('stadium')) {
+			value.set((Math.floor(value.value * 255 / 100) / 256) * 100);
```

Comment:
Should this be rounded? Is it already rounded?

Reply:
It gets rounded whenever accuracy is shown (Tested with 100% accurate move, actual number is 99.609375 and it showed 99.61% in the tooltip).

Reply:
Would prefer fewer digits, but I suppose that's good enough.